src/Aspire.Dashboard/Components/Controls/ApplicationName.razor.cs

```diff
@@ -12,6 +12,9 @@ public sealed partial class ApplicationName : ComponentBase, IDisposable
 {
     private CancellationTokenSource? _disposalCts;
 
+    [Parameter]
+    public string? AdditionalText { get; init; }
```

Comment:
The parameters in this control should be set, not init. Fix this one and the other parameters